src/main/java/io/github/dsheirer/gui/playlist/radioreference/LoginDialog.java

```diff
@@ -357,8 +391,10 @@ else if(rre.hasFault())
                                 }
                             }
 
-                            getTestPassIcon().setVisible(success);
-                            getTestFailIcon().setVisible(!success);
+                            // Set an error state
```

Comment:
The 3 getTestXXXIcon() lines probably need moved up into the catch block.  They're overwriting what gets set within the try block.

The getTestConnectionButton() can remain after the catch block.

Reply:
Oops, yeah, that needed to be moved. Fixed!